lib/jekyll-readme-index/generator.rb

```diff
@@ -26,15 +28,38 @@ def readme
     end
 
     def index?
-      (site.pages + site.static_files).any? { |file| file.url =~ INDEX_REGEX }
+      relative_index?(readme)
     end
 
     def page
+      relative_page(readme)
+    end
+
+    def readmes
+      site.static_files.each_with_object([]) do |file, memo|
```

Comment:
Could this be simplified to `site.static_files.select { |file| file.relative_path =~ READMES_REGEX }`?